client/src/components/organizations/OrganizationSwitchPopover.vue

```diff
@@ -2,61 +2,89 @@
 
 <template>
   <div class="popover-switch">
-    <ion-title class="popover-switch-title body">{{ $msTranslate('OrganizationSwitch.loggedInOrgs') }}</ion-title>
-    <ion-list class="organization-list">
-      <ion-item
-        class="organization-list__item body"
-        button
-        lines="none"
-        v-for="org in connectedOrgs"
-        :key="org.handle"
-        :disabled="org.active"
-        @click="onOrganizationClick(org)"
-      >
-        <div class="organization">
-          <ion-avatar class="organization-avatar">
-            <span>{{ org.id.substring(0, 2) }}</span>
-          </ion-avatar>
-          <div class="organization-text-content">
-            <ion-label class="organization-text-content__name">
-              <span class="body">
-                {{ org.id }}
-              </span>
-            </ion-label>
-            <ion-label class="organization-text-content__email">
-              <span class="body-sm">
-                {{ org.userLabel }}
-              </span>
-            </ion-label>
+    <div class="current-organization">
+      <div class="current-organization-content">
+        <ion-avatar class="organization-avatar">
+          <span v-if="!isTrialOrg">{{ currentOrg ? currentOrg.id.substring(0, 2) : '' }}</span>
+          <ms-image
+            v-else
+            :image="LogoIconGradient"
+            class="organization-avatar-logo"
+          />
+        </ion-avatar>
+        <ion-text class="organization-name title-h4">
+          {{ currentOrg?.id }}
+        </ion-text>
+      </div>
+    </div>
+    <div
+      class="connected-organization"
+      v-if="filteredOrgs.length > 0"
+    >
+      <ion-title class="connected-organization-title body">{{ $msTranslate('OrganizationSwitch.loggedInOrgs') }}</ion-title>
+      <ion-list class="organization-list">
+        <ion-item
+          class="organization-list__item body"
+          button
+          lines="none"
+          v-for="org in filteredOrgs"
+          :key="org.handle"
+          @click="onOrganizationClick(org)"
+        >
+          <div class="organization">
+            <ion-avatar class="organization-avatar">
+              <ms-image
+                v-if="org.trial"
+                :image="LogoIconGradient"
+                class="organization-avatar-logo"
+              />
+              <span v-else>{{ org.id.substring(0, 2) }}</span>
+            </ion-avatar>
+            <div class="organization-text-content">
+              <ion-label class="organization-text-content__name">
+                <span class="body">
+                  {{ org.id }}
+                </span>
+              </ion-label>
+              <ion-label class="organization-text-content__email body-sm">
+                <span v-if="expirationDuration && org.trial">
+                  {{ $msTranslate(formatExpirationTime(expirationDuration)) }}
+                </span>
```

Comment:
`expirationDuration` is set for the current org so this won't work since we're looking at non-current organizations here.

You can remove the `expirationDuration` variable entirely and set a function to check
```
function getExpirationDuration(device: AvailableDevice): Duration {
  return getDurationBeforeExpiration(device.createdOn);
}
```
, set the `device` and `trial` attributes for `ConnectedOrganization` as mandatory since you're checking it every time, and replace this commented call with 

```
<span v-if="org.trial">
  {{ $msTranslate(formatExpirationTime(getExpirationDuration(org.device))) }}
</span>
```

Reply:
Actually don't even bother with a new function
```
{{ $msTranslate(formatExpirationTime(getDurationBeforeExpiration(org.device.createdOn))) }}
```